nginx/nginx.conf

```diff
@@ -0,0 +1,30 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    set_real_ip_from 128.138.64.0/23;
+    set_real_ip_from 128.138.135.0/24;
+    real_ip_header X-Forwarded-For;
+    real_ip_recursive on;
```

Comment:
The stuff above tells nginx to use the X-Forwarded-For header to get the user's real IP address, but only trusting the whitelisted range of IPs to provide a correct X-Forwarded-For header. Without this, our IP whitelisting below won't work.